modules/network/src/testFixtures/java/org/apache/ignite/utils/ClusterServiceTestUtils.java

```diff
@@ -56,29 +57,47 @@ public class ClusterServiceTestUtils {
     private static final TestScaleCubeClusterServiceFactory SERVICE_FACTORY = new TestScaleCubeClusterServiceFactory();
 
     /** Registry initializers collected via reflection. */
```

Comment:
It may be worth mentioning that we are looking for methods here: `org.apache.ignite.network.serialization.MessageSerializationRegistry#registerFactory` ?

Reply:
I think this code is easy to read and is pretty self-explanatory

Reply:
It was not entirely obvious to me, I do not insist, but a mention would not be superfluous.

Reply:
Since we are going to get rid of this code anyway, I don't think it's that important

Reply:
Ok